src/components/CountryDetails.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import data from '../countries.json';
+
+function CountryDetails(props) {
+  const getCountry = (cca3) => {
+    const theCountry = (oneCountry) => {
+      return oneCountry.cca3 === cca3;
+    };
+    return data.find(theCountry);
+  };
+
+  const { params } = props.match;
+  const foundCountry = getCountry(params.cca3, 10);
+
+  return (
+    <div>
+      <h2>{foundCountry.name.common}</h2>
+      <ul>
+        <li>Capital: {foundCountry.capital[0]}</li>
+        <li>Area: {foundCountry.area} km2</li>
+        <li>
+          Borders:
```

Comment:
If you're going to make a nested list, you should have the `<ul>` nested inside your `<li>` element as well (check [this link](https://stackoverflow.com/questions/5899337/proper-way-to-make-html-nested-list#:~:text=Correct%3A%20as%20child,in%20which%20it%20is%20nested.) for reference)